recipes/expat/all/conanfile.py

```diff
@@ -52,12 +51,14 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
         self.cpp_info.name = "EXPAT"
         if self.settings.os == "Windows" and self.settings.build_type == "Debug":
-            self.cpp_info.libs = ["expatd"]
+            self.cpp_info.libs = ["libexpatd"]
         else:
-            self.cpp_info.libs = ["expat"]
+            self.cpp_info.libs = ["libexpat"]
```

Comment:
```suggestion
            self.cpp_info.libs = ["expat"]
```

https://github.com/conan-io/conan-center-index/pull/311#issuecomment-553620956